yml/cdb_latiss.yaml

```diff
@@ -1003,8 +1003,33 @@ tables:
     datatype: int
     description: Count of high signal-to-noise-ratio sources.
     ivoa:unit: count
+- name: exposure_quicklook
+  "@id": "#exposure_quicklook"
+  primaryKey:
+  - "#exposure_quicklook.day_obs"
+  - "#exposure_quicklook.seq_num"
+  constraints:
+  - name: fk_exposure_quicklook_day_obs_seq_num
+    "@id": "#exposure_quicklook.fk_exposure_quicklook_day_obs_seq_num"
+    "@type": ForeignKey
+    description: Quicklook day_obs must be an Exposure day_obs.
+    columns: ["#exposure_quicklook.day_obs", "#exposure_quicklook.seq_num"]
+    referencedColumns: ["#exposure.day_obs", "#exposure.seq_num"]
+  columns:
+  - name: day_obs
```

Comment:
Even though it isn't part of the primary key anymore, I'd still include the `exposure_id` (and give it a foreign key constraint).